$OpenBSD: patch-src_libraries_Box2D_Collision_Shapes_b2ChainShape_h,v 1.1 2019/10/18 15:11:09 cwen Exp $

ports-gcc fix for:
error: converting to 'bool' from 'std::nullptr_t' requires direct-initialization

Index: src/libraries/Box2D/Collision/Shapes/b2ChainShape.h
--- src/libraries/Box2D/Collision/Shapes/b2ChainShape.h.orig
+++ src/libraries/Box2D/Collision/Shapes/b2ChainShape.h
@@ -95,8 +95,8 @@ inline b2ChainShape::b2ChainShape()
 	m_radius = b2_polygonRadius;
 	m_vertices = NULL;
 	m_count = 0;
-	m_hasPrevVertex = NULL;
-	m_hasNextVertex = NULL;
+	m_hasPrevVertex = false;
+	m_hasNextVertex = false;
 }
 
 #endif
